src/test/java/hw2/Ex1/ExpectedConstant.java

```diff
@@ -0,0 +1,23 @@
+package hw2.Ex1;
+
+public interface ExpectedConstant {
+    String firstImageText = "To include good practices\n" +
+            "and ideas from successful\n" +
+            "EPAM project";
+     String secondImageText = "To be flexible and\n" +
+             "customizable";
+     String thirdImageText = "To be multiplatform";
+     String fourthImageText = "Already have good base\n" +
+             "(about 20 internal and\n" +
+             "some external projects),\n" +
+             "wish to get more…";
+
+     String mainHeader = "EPAM FRAMEWORK WISHES…";
+     String mainHeaderText = "LOREM IPSUM DOLOR SIT AMET, CONSECTETUR ADIPISICING ELIT, SED DO" +
+             " EIUSMOD TEMPOR INCIDIDUNT UT LABORE ET DOLORE MAGNA ALIQUA. UT ENIM AD MINIM VENIAM, " +
+             "QUIS NOSTRUD EXERCITATION ULLAMCO LABORIS NISI UT ALIQUIP EX EA " +
+             "COMMODO CONSEQUAT DUIS AUTE IRURE DOLOR IN REPREHENDERIT IN " +
+             "VOLUPTATE VELIT ESSE CILLUM DOLORE EU FUGIAT NULLA PARIATUR.";
+
+
+}
```

Comment:
What was changed in this file? I still see that **constant** names for follow Java Code Convention